helpers/src/main/java/com/linkedin/plugin/NGTestsBase.java

```diff
@@ -101,14 +101,18 @@ public T getTestServerAnnotation() {
       return conf;
     }
 
+    /**
+     * @deprecated Use dependency injection with Play 2.4 instead of plugins.
+     */
+    @Deprecated
     protected List<String> getPlugins(){
```

Comment:
This should be removed. Play 2.5 no longer supports Plugin.

Reply:
Leaving deprecated for 2.5 release; removing in 2.6.

Reply:
As @FranklinYinanDing points out support for Plugins is totally removed in 2.5. I think we should remove all references here too, including WithPlugins annotation, FakeApplicationFactoryArgs/Impl, tests, etc

Reply:
Okay, that's been cleaned out now. The only remaining vestige is a parameter on a deprecated constructor for the args builder, so folks not using `WithPlugins` don't get bitten by bincompat issues.